web/client/examples/viewer/plugins/mobile.jsx

```diff
@@ -121,6 +124,10 @@ module.exports = {
                                 crs={(props.mousePositionCrs) ? props.mousePositionCrs : props.map.projection} />
                         </Settings>
                     </Section>
+                    <Section eventKey="fullscreen" header={<Message msgId="settings-scr" />}>
+                        <h5>Fullscreen</h5>
+                        <FullScreen activeKey="1" />
```

Comment:
here you can set the fullscreen state (boolean) as an additional prop of the Fullscreen component